### ISSUE TITLE
pseudo publish py-0.2.0 for testing 

### ISSUE THREAD

**Issue by keighrim** [clamsproject/mmif#101]
Establish a way to continuously publish pre-release versions of py-0.2.0 to a test server (test.pypi or in-house nexus) so that they can be used in updating existing apps (and find more bugs in doing so).

**Comment by keighrim** [comment]
With fixes in 1170a79...eb546f2, one can simply run `make clean && make develop` to deploy a source distribution (with `devX` attached to the version, where X = latest dev version +1) to the lab nexus pypi repository, with an environment variable `$CLAMSUPLOADERPASSWORD` correctly set. 
To search or install from the lab pypi using pip use following commands; 

```
# search 
pip search --index http://morbius.cs-i.brandeis.edu:8081/repository/pypi-develop/pypi mmif-python
# install 
pip install --extra-index-url http://morbius.cs-i.brandeis.edu:8081/repository/pypi-develop/simple --trusted-host morbius.cs-i.brandeis.edu mmif-python
```

**Comment by keighrim** [comment]
Closed via #108 .
